diff --git a/src/helpers.ts b/src/helpers.ts
--- a/src/helpers.ts
+++ b/src/helpers.ts
@@ -4 +4 @@
-import * as querystring from 'query-string'
+import querystring from 'query-string'
diff --git a/src/internal/client.ts b/src/internal/client.ts
--- a/src/internal/client.ts
+++ b/src/internal/client.ts
@@ -13 +13 @@
-import * as qs from 'query-string'
+import qs from 'query-string'
diff --git a/dist/esm/helpers.mjs b/dist/esm/helpers.mjs
--- a/dist/esm/helpers.mjs
+++ b/dist/esm/helpers.mjs
@@ -3 +3 @@
-import * as querystring from 'query-string';
+import querystring from 'query-string';
diff --git a/dist/esm/internal/client.mjs b/dist/esm/internal/client.mjs
--- a/dist/esm/internal/client.mjs
+++ b/dist/esm/internal/client.mjs
@@ -10 +10 @@
-import * as qs from 'query-string';
+import qs from 'query-string';
diff --git a/src/notification.ts b/src/notification.ts
--- a/src/notification.ts
+++ b/src/notification.ts
@@ -18 +18 @@
-import jsonLineParser from 'stream-json/jsonl/Parser.js'
+import jsonLineParser from 'stream-json/jsonl/parser.js'
@@ -219 +219 @@
-        const asm = jsonLineParser.make()
+        const asm = jsonLineParser.asStream()
diff --git a/dist/esm/notification.mjs b/dist/esm/notification.mjs
--- a/dist/esm/notification.mjs
+++ b/dist/esm/notification.mjs
@@ -18 +18 @@
-import jsonLineParser from 'stream-json/jsonl/Parser.js';
+import jsonLineParser from 'stream-json/jsonl/parser.js';
@@ -179 +179 @@
-      const asm = jsonLineParser.make();
+      const asm = jsonLineParser.asStream();
diff --git a/dist/main/notification.js b/dist/main/notification.js
--- a/dist/main/notification.js
+++ b/dist/main/notification.js
@@ -7 +7 @@
-var _Parser = require("stream-json/jsonl/Parser.js");
+var _Parser = require("stream-json/jsonl/parser.js").default;
@@ -199 +199 @@
-      const asm = _Parser.make();
+      const asm = _Parser.asStream();
